packages/mdc-icon-button/_mixins.scss

```diff
@@ -79,7 +81,28 @@
   }
 }
 
-@mixin mdc-icon-button-size($width, $height: $width, $padding: max($width, $height)/2, $query: mdc-feature-all()) {
+@mixin mdc-icon-button-size($size, $query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-feature-targets($feat-structure) {
+    width: $size;
+    height: $size;
+    padding: $size / 4;
```

Comment:
I would remove the padding or set it to `0` and use flex box to center align the icon.

Reply:
Done. By doing this however some of the screenshot tests changed.